static/gsApp/components/ai/AiSetupDataConsent.tsx

```diff
@@ -23,7 +33,25 @@ function AiSetupDataConsent({groupId}: AiSetupDataConsentProps) {
   const api = useApi({persistInFlight: true});
   const organization = useOrganization();
   const queryClient = useQueryClient();
-  const {data: autofixSetupData} = useAutofixSetup({groupId});
+  const {data: autofixSetupData, hasAutofixQuota, refetch} = useAutofixSetup({groupId});
+  const navigate = useNavigate();
+  const subscription = useSubscription();
+
+  const trial = getProductTrial(
+    subscription?.productTrials ?? null,
+    DataCategory.SEER_AUTOFIX
+  );
+
+  const orgHasAcknowledged = autofixSetupData?.setupAcknowledgement.orgHasAcknowledged;
+  const shouldShowBilling =
+    organization.features.includes('seer-billing') && !hasAutofixQuota;
+  const canStartTrial = Boolean(trial && !trial.isStarted);
+  const hasSeerButNeedsPayg =
+    shouldShowBilling && organization.features.includes('seer-added');
+
+  const isTouchCustomer = subscription?.type === BillingType.INVOICED;
```

Comment:
we likely need to special case for sponsored customers too since they won't have access to seer at all

Reply:
It was decided sponsored orgs won't be able to use Seer but we should double check the expected behavior for them in the project channel